### PR TITLE
fix: corrected string formatting example

### PR DESCRIPTION
**Pull request by ajejoseph22**
# Changelog

Corrected string formatting example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -606,14 +606,14 @@ Now, let's quickly look at some more examples. Here we will try to calculate a p
...
 ...
 ```
 
 ```bash
 $ go run main.go
-58.181818
+77.777778
 ```
 
 Let's say we want just `58.18` which is 2 points precision, we can do that as well by using `.2f`.
```
Comment (karanpratapsingh) [inline README.md]: Can we update `58.18` to `77.78` here?

Comment (ajejoseph22) [reply README.md]: Ah, missed that

Comment (karanpratapsingh) [reply README.md]: No worries

Comment (ajejoseph22) [reply README.md]: It's done. @karanpratapsingh

Comment (karanpratapsingh) [reply README.md]: Thanks!

**Review by karanpratapsingh — APPROVED** [review]
Thank you for the fix @ajejoseph22!

**Pull request merged**